docs/pages/architecture/tls-routing.mdx

```diff
@@ -85,8 +85,8 @@ which can be used as a `ProxyCommand`.
 Similarly to `tsh ssh`, `tsh proxy ssh` establishes a TLS tunnel to Teleport
 proxy with `teleport-ssh-proxy` ALPN protocol, which `ssh` then connects over.
 
-See [OpenSSH client](../server-access/guides/openssh.mdx#use-the-openssh-client-to-access-teleport-nodes)
-guide for details on how it's configured.
+See [OpenSSH client](../server-access/guides/openssh.mdx) guide for details on
```

Comment:
Small grammar nit: should this be "See **the** OpenSSH client guide" ?

Reply:
Yes, changed!